templates/case/_case_show/_resident_and_rent_board_shareds.html.twig

```diff
@@ -21,7 +21,12 @@
         <div class="row mt-3">
             <div class="col">
                 <fieldset>
-                    <legend>{% trans %}Bringer{% endtrans %}</legend>
+                    <legend>
+                        {% trans %}Bringer{% endtrans %}
+                        {% if case.bringerIsUnderAddressProtection %}
+                            <p id="bringerIsUnderAddressProtectionCaseShow">!{% trans %}Is under address protection{% endtrans %}!</p>
```

Comment:
See previous comment on “!” and translatable text.

Reply:
Updated.